__tests__/app.test.js

```diff
@@ -161,3 +161,75 @@ describe("GET /api/reviews/:review_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/reviews/:review_id/comments", () => {
+  test("201: responds with the posted comment", () => {
+    const newComment = { username: "mallionaire", body: "I love this game!" };
+    return request(app)
+      .post("/api/reviews/4/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toMatchObject(
+          expect.objectContaining({
+            comment_id: 7,
+            body: "I love this game!",
+            votes: 0,
+            author: "mallionaire",
+            review_id: 4,
+            created_at: expect.any(String),
+          })
+        );
+      });

```

Comment:
Great testing here Mohamed! Could you just add one more 201 test which ignores unnecessary properties? I.e. if you include an extra property apart from username and body as the newComment, it should ignore it. 👍